### PR TITLE
Release fixes for rebuild

### PR DESCRIPTION
**Pull request by ericstj**
We're planning to repackage Microsoft.NETCore.Runtime.CoreCLR and Microsoft.NETCore.Jit with these fixes.  Ensure we have them in the release branch.

/cc @weshaggard @gkhanna79

### CODE REVIEW

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <!-- Declare a runtime dependency on the win8-arm CoreCLR built using the TFS builds -->
     <RuntimeDependency Include="runtime.win8-arm.Microsoft.NETCore.Runtime.CoreCLR">
       <TargetRuntime>win8-arm</TargetRuntime>
-      <Version>1.0.2-$(ExternalExpectedPrerelease)</Version>
```
Comment (gkhanna79) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: Would we need to add this back after the fix is out?

Comment (ericstj) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: Not in release.

**Comment by weshaggard** [comment]
LGTM.

**Comment by ericstj** [comment]
@dotnet-bot test  Ubuntu x64 Checked Build and Test

**Comment by weshaggard** [comment]
Talked with @gkhanna79 offline and we need this change for servicing so merging.

**Pull request merged**